doc/conftest.py

```diff
@@ -47,12 +47,14 @@ def setup_rcv1():
 
 def setup_twenty_newsgroups():
     data_home = get_data_home()
-    if not exists(join(data_home, '20news_home')):
+    if not exists(join(data_home, "20news-bydate_py3.pkz")):
```

Comment:
is this python3 specific? Maybe the easiest way would be to try and load it and catch the except?

Reply:
Changed to use the pickle filepath specific to the python version of the test runner.